### PR TITLE
View individual event

### PR DESCRIPTION
**Pull request by cheljoh**
Add functionality for viewing specific events. Group tickets for listing. Presently, seat is not presented in view, only row. May need to change in future.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -35,13 +36,11 @@
...
-  concerts.events.create(
-    name: "Muppet Rock",
-    venue_id: venue2.id,
-    image: open("https://s3.amazonaws.com/ticket-cacher/categories/muppets_rocking.jpg"),
-    time: "November 20, 2020 18:00", status: "active")
-end
+concerts.events.create(
+  name: "Muppet Rock",
+  venue_id: venue2.id,
+  image: open("https://s3.amazonaws.com/ticket-cacher/categories/muppets_rocking.jpg"),
```
Comment (cheljoh) [inline db/seeds.rb:41]: Will ignore hound this time, don't want to split up URL.

File: spec/features/user/user_views_events_by_category_spec.rb [hunk 2]
```diff
@@ -8,9 +8,9 @@
     event2 = make_events[:event2]
     event3 = make_events[:event3]
 
-    visit root_path
+    visit "/festivals"
 
-    click_on "festivalz"
+    # add css image
```
Comment (erinnachen) [inline spec/features/user/user_views_events_by_category_spec.rb:13]: is there something you're missing here?

File: spec/features/user/user_views_a_specific_event_spec.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      expect(page).to have_content("price: $9")
+      expect(page).to have_content("row: 5")
+      expect(page).to have_content("quantity: 2")
+    end
+
+    within("#listing-#{listing_2.id}") do
+      expect(page).to have_content("row: 7")
+      expect(page).to have_content("price: $10")
+      expect(page).to have_content("quantity: 1")
+    end
```
Comment (erinnachen) [inline spec/features/user/user_views_a_specific_event_spec.rb:38]: maybe add this: expect(page).to_not have_content("quantity: 2")

File: app/views/shared/_user_functions.html.erb [hunk 4]
```diff
@@ -5,25 +5,22 @@
...
     <li class="name-in-dropdown">
-      <%= link_to "hipster pop-ups", "#" %>
+      <%= link_to "conventions", "/conventions" %>
     </li>
     <li class="name-in-dropdown">
-      <%= link_to "conventions", "#" %>
-    </li>
-    <li class="name-in-dropdown">
-      <%= link_to "comedy", "#" %>
+      <%= link_to "comedy", "comedy" %>
```
Comment (erinnachen) [inline app/views/shared/_user_functions.html.erb]: should there be a slash here ("/comedy")

File: app/models/event.rb [hunk 5]
```diff
@@ -1,6 +1,8 @@
 class Event < ActiveRecord::Base
   belongs_to :category
   belongs_to :venue
+  has_many :listings
+
```
Comment (erinnachen) [inline app/models/event.rb:5]: is this relationship tested?

File: app/models/event.rb [hunk 6]
```diff
@@ -1,6 +1,8 @@
...
   # has_many :order_products
   # has_many :orders, through: :order_products
```
Comment (erinnachen) [inline app/models/event.rb:8]: we should take out these comments

**Comment by damwhit** [comment]
I like it.  Glad you went back to test more models.

**Pull request merged**